account_invoice_report_hls/report/invoice_delivery.xml

```diff
@@ -0,0 +1,236 @@
+                        alt="Logo"
+                    />
+                </div>
+            </div> -->
+            <!-- <div
+                t-if="company.logo or company.report_header"
+                class="row zero_min_height"
+            >
+                <div class="col-12">
+                    <div style="border-bottom: 1px solid black;" />
+                </div>
+            </div> -->
+            <!-- <div class="row">
+                <div t-if="doc_type != 'invoice'" class="col-12 text-right">
+                    <strong t-field="o.invoice_id.date_invoice" t-options='{"widget": "date"}' />
+                </div>
+            </div> -->
+        </div>
+
+        <div class="article o_report_layout_standard" t-att-data-oe-model="o and o._name" t-att-data-oe-id="o and o.id" t-att-data-oe-lang="o and o.env.context.get('lang')">
+            <t t-call="web.address_layout"/>
+            <t t-raw="0"/>
+        </div>
+
+        <div class="footer o_standard_footer">
+            <div class="text-center" style="border-top: 1px solid black;">
+                <ul class="list-inline mb4">
+                    <div t-field="company.report_footer" />
+                </ul>
+                <div t-if="report_type == 'pdf'" class="text-muted">
+                    Page: <span class="page" /> / <span class="topage" />
+                </div>
+            </div>
+        </div>
+    </template>
+
+
+    <template id="report_invoice_document">
+        <t t-call="account_invoice_report_hls.external_layout_invoice_hls">
+            <t t-set="o" t-value="o.with_context(lang=lang)" />
+            <div class="page">
+                <h2 class="text-center">
+                    <span t-if="o.invoice_id.type == 'out_invoice' and o.invoice_id.state in ('open', 'in_payment', 'paid')">Invoice</span>
+                    <span t-if="o.invoice_id.type == 'out_invoice' and o.invoice_id.state == 'draft'">Draft Invoice</span>
+                    <span t-if="o.invoice_id.type == 'out_invoice' and o.invoice_id.state == 'cancel'">Cancelled Invoice</span>
+                    <span t-if="o.invoice_id.type == 'out_refund'">Credit Note</span>
+                    <span t-if="o.invoice_id.type == 'in_refund'">Vendor Credit Note</span>
+                    <span t-if="o.invoice_id.type == 'in_invoice'">Vendor Bill</span>
+                    <span t-field="o.invoice_id.doc_title" />
+                </h2>
+                <div class="row" style="padding-top: 20px;">
+                    <div class="col-6">
+                        <div>
+                            <address t-field="o.invoice_id.partner_id" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' />
+                        </div>
+                        <div style="padding-top: 60px;">
+                            <div>
+                                <p>
+                                    <div>平素は格別のご高配に賜り熱く御礼申し上げます。</div>
```

Comment:
```suggestion
                                    <div>平素は格別のご高配に賜り厚く御礼申し上げます。</div>
```
We should ideally make this sentence configurable in company form, although it is not a must at this stage.